packages/client/src/BrubeckNode.ts

```diff
@@ -165,4 +165,30 @@ export default class BrubeckNode implements Context {
             this.debug('publishToNode << %o', streamMessage.getMessageID())
         }
     }
+
+    openPublishProxyConnectionOnStream(streamId: string, streamPartition: number | undefined, nodeId: string): void | Promise<void> {
```

Comment:
The public methods could use a `SPID` instance to wrap `streamId`+`streamPartittion`. And rename to `openPublishProxyConnectionOnStreamPartition` as it always targets only one stream partition, not the whole stream.

Similar fix also for line 183, and for new methods in `Publisher`